### DISCUSSION TITLE
java8/stream

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#215] [Announcements]
# java8/stream

本文详细介绍了Java 8引入的Stream流，阐述了Stream流的特点和用法。通过实际的代码示例，展示了如何使用Stream流对集合进行高效、简洁的操作。学习本文，让您快速掌握Java 8 Stream流的实践技巧，体验流式编程带来的编程乐趣。

https://tobebetterjavaer.com/java8/stream.html

**Comment by congshuang** [comment]
作者写的真心不错,这样幽默风趣的风格,真心喜欢
